.gitignore

```diff
@@ -0,0 +1 @@
+.DS_Store
```

Comment:
This file will tell git to not track some files.
In this case, I'm only adding `.DS_Store` as it's a macOS file control and it's not useful for any other developer.